components/cookie/banner/package.json

```diff
@@ -0,0 +1,21 @@
+{
+  "name": "@schibstedspain/sui-cookie-banner",
+  "version": "1.0.0",
+  "description": "",
+  "main": "lib/index.js",
+  "scripts": {
+    "build": "rm -Rf ./lib && mkdir -p ./lib && npm run babel && npm run sass",
+    "babel": "../../../node_modules/.bin/babel --presets schibsted-spain ./src --out-dir ./lib",
+    "sass": "../../../node_modules/.bin/cpx \"./src/**/*.scss\" ./lib",
+    "preversion": "echo \"preversion\"",
+    "version": "npm run build",
+    "postversion": "git add ${PWD} && git commit -m \"release(cookie/banner): v$(node -p -e \"require('./package.json').version\")\" && git push origin master && npm publish --access=public"
```

Comment:
`preversion`, `version` and `postversion` is not needed anymore with suistudio v4.